pkg/cri/server/restart.go

```diff
@@ -146,6 +166,22 @@ func (c *criService) recover(ctx context.Context) error {
 		}
 	}
 
+	for _, sb := range c.sandboxStore.List() {
+		sb := sb
+		go func() {
```

Comment:
Is it bugfix?

Reply:
I think it can be considered as a bugfix, before this PR, the recover.go in podsandbox will handle the task exit and update the sandbox in the cri cache. Now as in this pr we removed the sandboxStore in podsandbox controller, it can not update the sandbox in cri cache anymore. so we have to call Wait of sandbox and startSandboxExitMonitor here.
I think it is also neccessary if we have more sandbox controllers other than podsandbox. we have to call Wait of sandbox and monitor it by ourselves.